lib/tasks/csv.rake

```diff
@@ -0,0 +1,37 @@
+desc "Scrape companies"
+
+task :make_companies => :environment do 
+  require 'open-uri'
+  require 'csv'
+
+  file_path = "lib/tasks/district_eleven.csv"
+
+  # file_data = open(file_path)
+
+  CSV.foreach(file_path, :headers => true) do |row|
+    
+    # puts "#{sn}"
+    # puts "#{project_name}"
+    # puts "#{street_name}"
+    # puts "#{type}"
+    # Companyname.create(name: "#{name}", symbol: "#{symbol}")
+    
+    Listing.create(
+      :sn => row["sn"],
+      :project_name => row["project_name"],
```

Comment:
Those listings are all the past transactions for given projects. Wouldn't it be useful to create a Project model, that has_many listings?